lib/gui/pages/main/controllers/flash.js

```diff
@@ -26,10 +26,50 @@ module.exports = function(
   FlashErrorModalService,
   ErrorService,
   OSNotificationService,
-  OSWindowProgressService
+  OSWindowProgressService,
+  DriveInfoModalService
```

Comment:
Make sure the main page declares `DriveInfoModal` as a dependency before using it.